test/Python/src/docker/FilesPropertyContainerized.lf

```diff
@@ -0,0 +1,34 @@
+target Python {
+    files: "../include/hello.py",
+    docker: true
+};
+
+preamble {=
+    try:
+        import hello
+    except:
+        request_stop()
```

Comment:
I like this approach, although I'm not sure how `request_stop()` would behave at the time of execution of this code, especially in federated execution, since the program might not be fully initialized yet.

Reply:
good point. I am down to settle for this approach for now, since the purpose of the test is to make sure files specified in the "files" target property are also available within the container during build time.